app/javascript/packs/admin.js

```diff
@@ -30,3 +30,27 @@ Rails.href = function Rails_href_override(element) {
 };
 
 Rails.start();
+
+
+document.addEventListener('click', function (event) {
+  // If the clicked element doesn't have the right selector, bail
+  if (event.target.matches('#sidebar-toggle, #sidebar-toggle *, i')) {
```

Comment:
This is fine for now, but down the road we should allow the accordion feature as part of a `class` rather than an `id`.

Reply:
Agreed, we could fairly easily extract this into a function but I decided not to do that for now 👍